presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -124,10 +145,12 @@
 public class IcebergMetadata
         implements ConnectorMetadata
 {
+    public static final String DEPENDS_ON_TABLES = "dependsOnTables";
```

Comment:
This should probably start with `presto` since it is Presto specific

Reply:
Hmm, not really presto specific. This property says that the current table was computed using these other tables with corresponding snapshot-ids. I would rather leave it as is. If you feel strongly, please let me know and I will change it.